exports/ystdlib-py/README.md

```diff
@@ -0,0 +1,82 @@
+# ystdlib-py
+
+Python utilities developed and used at YScope.
+
+## Modules
+
+How to get started with the modules inside ystdlib-py.
+
```

Comment:
```suggestion
```

Although we shouldn't have two consecutive headings, I think this sentence is a bit redundant.